docker/lambda/s3.ts

```diff
@@ -0,0 +1,50 @@
+import { GetObjectCommand, PutObjectCommand, S3Client } from '@aws-sdk/client-s3'
+import { fromContainerMetadata } from '@aws-sdk/credential-providers'
```

Comment:
```suggestion
import { fromInstanceMetadata} from '@aws-sdk/credential-providers'
```

多分 `fromInstanceMetadata` だと思うのですが、一旦こっちに変えてもらって、後々調整して動かなければまた考えようかと思います。

Reply:
間に合ったらデプロイまで試しますが、間に合わなければ別 PR で調整作業します。

Reply:
fromInstanceMetadataに変えておきました

Reply:
後でやりますが、 `fromEnv()` が正解でした…
https://github.com/violet-ts/violet-infrastructure/issues/4